src/common/notification/models/response.ts

```diff
@@ -3,10 +3,10 @@ export const notificationData = {
     {
       id: 43,
       role: 'Customer',
-      entity_type: 'Order',
-      entity: 'OrderCreated',
+      entity_type: 'Genetic Testing Order',
+      entity: 'Order Created',
       description:
-        'Congrats! Your requested test for 0xd908eb33... has been submitted.',
+        "You've successfully submitted your requested test for 0xd908eb33... .",
```

Comment:
Hardcoded?

Reply:
before refactor, this file used by swagger responce in debio-backend.
now has been deleted.